app/middlewares/fields_validation.js

```diff
@@ -8,8 +8,8 @@ const buildErrorMessage = errors =>
     .map(error => error.msg)
     .join('; ');
 
-exports.fieldsValidation = validations => (req, res, next) => {
-  Promise.all(validations.map(validation => validation.run(req)))
+exports.fieldsValidation = (...validations) => (req, res, next) => {
+  Promise.all(validations.flat().map(validation => validation.run(req)))
```

Comment:
There is no need to run the schemas by hand. `checkSchema` runs the validations and let you check the errors with the function `validationResult`.

Reply:
If I don't run validations, when checking `validationsResult`, errors is always empty

Reply:
The idea of this method is to short-circuit the middleware chain calling the error middleware when validation result is fail.
If I don't do this, I will always repeat `if(!errors.isEmpty()) return next(<error>)` on all routes.